src/lib.rs

```diff
@@ -284,7 +326,7 @@ impl<E: core::fmt::Debug, ODO: OpenDrainOutput<Error = E>> OneWire<ODO> {
 
     pub fn reset_select_read_only(
         &mut self,
-        delay: &mut impl DelayUs<u16>,
+        delay: &mut impl DelayNs,
```

Comment:
I needed to get rid of the second select in this function to get correct readings: https://github.com/kellerkindt/onewire/pull/18/commits/32c371a296068865d86409b220e5014d4fe98bc4

Reply:
Should be fine, now that #16 is merged?

Reply:
Yes.